awx/ui_next/src/screens/Template/Survey/SurveyListItem.jsx

```diff
@@ -27,8 +28,20 @@ const Button = styled(_Button)`
   padding-bottom: 0;
   padding-left: 0;
 `;
+const Required = styled.span`
+  color: red;
+  margin-left: 5px;
+`;
+const Chip = styled(_Chip)`
+  margin-right: 5px;
+`;
```

Comment:
Can you use a ChipGroup to contain these instead of adding custom styles? It should take care the spacing for you